src/Components/Header/Header.jsx

```diff
@@ -1,48 +1,56 @@
 import "./Header.css";
 // import { useEcom } from "../../Context/useEcomReducer.context";
-// import { Link } from "react-router-dom";
+import { Link } from "react-router-dom";
+import { LandingPage } from "../../Pages/LandingPage/LandingPage";
 // import { ProfilePage } from "../../Profile/ProfilePage";
 // import { WishlistPage } from "../../WIshlist/WishlistPage";
 // import { CartPage } from "../../Cart/CartPage";
+import { useLocation } from "react-router-dom";
 
 export const Header = () => {
-  const { forLength } = useEcom();
+  // const { forLength } = useEcom();
+
+  const currentPath = useLocation();
 
   return (
     <nav>
       <div className="navbar">
-        <h1 className="name">
-          <i className="fas fa-bolt"></i> Stormpoint
-        </h1>
-        <input className="header-input" placeholder="Search" type="text" />
+        <Link to="/" element={<LandingPage />}>
+          <h1 className="name">
+            <i className="fas fa-bolt"></i> Stormpoint
+          </h1>
+        </Link>
+        {currentPath.pathname === "/Products" && (
+          <input className="header-input" placeholder="Search" type="text" />
+        )}
         <div className="margin-left-auto">
           <div className="icon-container">
             <div className="badge">
-              <Link to="/Profile" element={<ProfilePage />}>
-                <i className="fa-solid fas fa-user"></i>
-              </Link>
+              {/* <Link to="/Profile" element={<ProfilePage />}> */}
+              <i className="fa-solid fas fa-user"></i>
+              {/* </Link> */}
             </div>
             <div className="badge">
-              <Link to="/Wishlist" element={<WishlistPage />}>
-                <i className="far fa-heart"></i>
-              </Link>
-              <div className="number">{forLength.wishlistLength}</div>
+              {/* <Link to="/Wishlist" element={<WishlistPage />}> */}
+              <i className="far fa-heart"></i>
+              {/* </Link> */}
+              <div className="number">1</div>
```

Comment:
suggestion - remove commented code

Reply:
thanks for pointing it out as i have created this site in other repo few things are commented out. these will be fixed soon